sql/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -1818,4 +1820,23 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
       assert(textResult==expected)
     }
   }
+
+  it ("Should pass ST_GeometricMedian") {
+    val geomTestCases = Map(
+      ("'MULTIPOINT((10 40), (40 30), (20 20), (30 10))'", 1e-15) -> "'POINT(22.5 21.25)'",
+      ("'MULTIPOINT((0 0), (1 1), (2 2), (200 200))'", 1e-6) -> "'POINT (1.9761550281255005 1.9761550281255005)'",
+      ("'MULTIPOINT ((0 0), (10 1), (5 1), (20 20))'", 1e-15) -> "'POINT (5 1)'",
+      ("'MULTIPOINT ((0 -1), (0 0), (0 0), (0 1))'", 1e-6) -> "'POINT (0 0)'",
+      ("'POINT (7 6)'", 1e-6) -> "'POINT (7 6)'",
+      ("'MULTIPOINT ((12 5),(62 7),(100 -1),(100 -5),(10 20),(105 -5))'", 1e-15) -> "'POINT(84.21672412761632 0.1351485929395439)'",
```

Comment:
Scala compilation is failed here: `error file=/home/runner/work/sedona/sedona/sql/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala message=expected start of definition, but was Token(VAL,val,85798,val)`

This is likely caused by this comma `,`. Please remove the last comma which is not needed.